src/silo/config/util/abstract_config_source.cpp

```diff
@@ -0,0 +1,34 @@
+#include "silo/config/util/abstract_config_source.h"
+
+#include <string>
+
+#include <boost/algorithm/string/join.hpp>
+#include <boost/lexical_cast.hpp>
+
+std::string silo::config::AbstractConfigSource::Option::toString() const {
+   return boost::join(access_path, ",");
+}
+
+std::optional<int32_t> silo::config::AbstractConfigSource::getInt32(const Option& option) const {
+   const auto string_value = getString(option);
+   if (string_value == std::nullopt) {
+      return std::nullopt;
+   }
+   try {
+      return boost::lexical_cast<int32_t>(*string_value);
+   } catch (boost::bad_lexical_cast&) {
+      return std::nullopt;
```

Comment:
Or should we rather throw in this case? Spotting those type errors can be quite hard when SILO silently ignores them.